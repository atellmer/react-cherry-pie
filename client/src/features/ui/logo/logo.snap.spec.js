import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import Logo from './index';


test('Logo (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<Logo/>);

  expect(tree).toMatchSnapshot();
});
