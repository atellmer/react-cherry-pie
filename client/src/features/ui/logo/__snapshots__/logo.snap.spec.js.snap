// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest: TmLogo (Snapshot) 1`] = `
<div
  className={undefined}
/>
`;
